Pages/Index.cshtml

```diff
@@ -154,8 +154,8 @@
                     <thead>
                         <tr>
                             <th scope="col">Property Type</th>
-                            <th scope="col">Project Categaroy</th>
-                            <th scope="col">Project cost</th>
+                            <th scope="col">Project Category</th>
```

Comment:
Renamed the Column which had incorrect names. Reduces technical debt by storing appropriate information.

Sources
https://www.w3.org/TR/dwbp/

Reply:
Good.